libr/cons/dietline.c

```diff
@@ -1182,16 +1204,7 @@ R_API const char *r_line_readline_cb(RLineReadCallback cb, void *user) {
 		case 24:// ^X -- do nothing but store in prev = *buf
 			break;
 		case 25:// ^Y - paste
-			if (I.clipboard != NULL) {
-				I.buffer.length += strlen (I.clipboard);
-				// TODO: support endless strings
-				if (I.buffer.length < R_LINE_BUFSIZE) {
-					I.buffer.index = I.buffer.length;
-					strcat (I.buffer.data, I.clipboard);
-				} else {
-					I.buffer.length -= strlen (I.clipboard);
-				}
-			}
+			yank();
```

Comment:
add missing space before (

Reply:
btw yank means copy, aka put buffer into clipboard. this case is doing a paste, not a yank

Reply:
In emacs/readline terminology "yank" is equivalent to "paste". I think "yank" as copy is something vim did. I had the impression that the existing codebase was using the emacs/readline terminology per [this comment](https://github.com/radare/radare2/blob/eb7deb281df54771fb8ecf5890dc325a7d22d3e2/libr/cons/dietline.c#L712), but I would be happy to go back and change it if you would like.